scipy/conftest.py

```diff
@@ -117,6 +117,9 @@ def check_fpu_mode(request):
     try:
         import array_api_strict
         xp_available_backends.update({'array_api_strict': array_api_strict})
+        array_api_strict.set_array_api_strict_flags(
+            api_version='2023.12'
+        )
```

Comment:
Locally this gave me an error before I upgraded to 2.0. Might want to let everyone know to upgrade.

Reply:
we could add a version check and a more helpful error message here, and add a lower bound to `pyproject.toml`?

Reply:
Ah, sounds good.

Reply:
Okay, I tried to do this, please test